mtt/src/mtt.rs

```diff
@@ -534,6 +558,14 @@ impl Mtt {
         }
     }
 
+    // Test only NOP validation
+    #[cfg(test)]
+    fn validate_mtt(&self, l2_base_addr: u64) -> Result<()> {
+        let _l2_dir = self.inner.lock();
+        SupervisorPageAddr::new(RawAddr::supervisor(l2_base_addr)).ok_or(Error::InvalidAddress)?;
```

Comment:
you could just return `Ok(())` rather than have what's effectively a no-op here.